assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -33,6 +34,11 @@ if (!intCmp) {
   store.dispatch(setIntCmp(intCmp));
 }
 
+const participations = store.getState().common.abParticipations;
+if (participations.annualContributions) {
+  trackOphan('annualContributions', participations.annualContributions);
```

Comment:
In this line, Are you tracking both test's variant or just the annual one? If you are tracking both, probably it would be better to name it `addingAnnualContribution` in order to increase readability.